activerecord/test/cases/adapters/postgresql/uuid_test.rb

```diff
@@ -120,6 +120,13 @@ def test_invalid_uuid_dont_match_to_nil
     assert_empty UUIDType.where(guid: "foobar")
   end
 
+  def test_case_insensitive_changes_to_uuids_do_not_mark_dirty
+    uuid = "AbCd-0123-4567-89Ef-DeAd-BeEf-0101-1010"
+    model = UUIDType.create!(guid: uuid)
+    model.guid = model.guid.swapcase
```

Comment:
I believe this is actually testing something subtly different than `model.guid = uuid.swapcase` since `model.guid` should be returning the already-cast value.  However, I think I like this better.  It better communicates that changing the value's case has no effect.

And this way, you can just get rid of the `uuid` variable:
```ruby
model = UUIDType.create!(guid: "abcd-0123-4567-89ef-dead-beef-0101-1010")
model.guid = model.guid.swapcase
assert_not model.changed?
```

The same goes for the `macaddr` test.

Reply:
I have removed the unnecessary variables.